examples/apps/custom-dns/all.yaml

```diff
@@ -0,0 +1,88 @@
+apiVersion: dns.gardener.cloud/v1alpha1
```

Comment:
Let's remove this one too. Is not universal and will not work for most users, as it has our domain names hardcoded